waspc/cli/src/Wasp/Cli/Util/PathArgument.hs

```diff
@@ -16,11 +17,18 @@ import System.Directory (makeAbsolute)
 -- in the meantime; so we make these types opaque until we have access to the IO
 -- monad.
 
-newtype FilePathArgument = FilePathArgument FilePath
-  deriving (Show, Eq)
+newtype FilePathArgument = FilePathArgument FilePath deriving (Show, Eq)
 
-filePathReader :: ReadM FilePathArgument
-filePathReader = FilePathArgument <$> str
```

Comment:
I removed the `filePathReader` because, while implementing `DirPathArgument`, I realized that the default parser accepts any `IsString` instance directly, and it's better declared that way.